solr/solrj/src/java/org/apache/solr/client/solrj/impl/Http2SolrClient.java

```diff
@@ -1020,12 +1025,6 @@ public Http2SolrClient build() {
       if (cookieStore == null) {
         cookieStore = getDefaultCookieStore();
       }
-      if (idleTimeoutMillis == null || idleTimeoutMillis <= 0) {
-        idleTimeoutMillis = (long) HttpClientUtil.DEFAULT_SO_TIMEOUT;
-      }
-      if (connectionTimeoutMillis == null) {
-        connectionTimeoutMillis = (long) HttpClientUtil.DEFAULT_CONNECT_TIMEOUT;
-      }
```

Comment:
No longer need such fields on the SolrClient since they are the responsibility of the HttpClient, thus they were redundant here and misleading (setting them changes nothing) IMO